source/upgrade.txt

```diff
@@ -23,41 +20,51 @@ Upgrade Driver Versions
 Overview
 --------
 
-On this page, you can learn how to upgrade your driver to a new version. This page also
-includes the changes you must make to your application to upgrade your driver without
-losing functionality, if applicable.
-
-Deprecation
------------
-
-Before we remove a feature in a major release, PyMongo's maintainers make an
-effort to release at least one minor version that *deprecates* it. We add
-"**DEPRECATED**" to the feature's documentation, and update the code to raise a
-``DeprecationWarning``. You can ensure your code is future-proof by running
-your code with the latest PyMongo release and looking for ``DeprecationWarning``
-messages.
-
-The interpreter silences ``DeprecationWarning`` messages by default. For example, the
-following code uses the deprecated ``insert`` method but does not raise any
-warning:
-
-.. code-block:: python
+This page describes the changes you must make to your application
+when you upgrade to a new version of {+driver-short+}.
 
-   # "insert.py" (with PyMongo 3.X)
-   from pymongo import MongoClient
+.. important:: 
+  
+   This guide includes breaking changes only to {+driver-short+} versions after v4.0.
+   If you're upgrading from {+driver-short+} v2 or v3, see the
+   `{+driver-short+} 4 Migration Guide. <https://pymongo.readthedocs.io/en/latest/migrate-to-pymongo4.html>`__
+
+Before you upgrade, perform the following actions:
+
+- Ensure the new {+driver-short+} version is compatible with the MongoDB Server versions
```

Comment:
```suggestion
- Ensure the new {+driver-short+} version is compatible with the MongoDB Server version
```

Reply:
I thought about this when writing it, but I think you could have a single application (or multiple instances of the same application) connecting to multiple deployments